templates/frontend/pages/editorialMasthead.tpl

```diff
@@ -15,38 +15,55 @@
 
 	<h1>{translate key="common.editorialMasthead"}</h1>
 	{foreach from=$mastheadRoles item="mastheadRole"}
-		{if $mastheadRole->getRoleId() != \PKP\security\Role::ROLE_ID_REVIEWER}
-			{if array_key_exists($mastheadRole->getId(), $mastheadUsers)}
-				<h2>{$mastheadRole->getLocalizedName()|escape}</h2>
-				<ul>
-				{foreach from=$mastheadUsers[$mastheadRole->getId()] item="mastheadUser"}
-					<li>
-						{$mastheadUser['user']->getFullName()|escape},
-						{$mastheadUser['user']->getLocalizedData('affiliation')|escape},
-						{$mastheadUser['dateStart']}
-						{if $mastheadUser['user']->getData('orcid')}
-							<span class="orcid">
-								{if $mastheadUser['user']->getData('orcidAccessToken')}
-									{$orcidIcon}
-								{/if}
-								<a href="{$mastheadUser['user']->getData('orcid')|escape}" target="_blank">
-								{$mastheadUser['user']->getData('orcid')|escape}
-							</a>
-							</span>
-						{/if}
-					</li>
-				{/foreach}
-				</ul>
-			{/if}
+		{if array_key_exists($mastheadRole->getId(), $mastheadUsers)}
+			<h2>{$mastheadRole->getLocalizedName()|escape}</h2>
+			<ul>
+			{foreach from=$mastheadUsers[$mastheadRole->getId()] item="mastheadUser"}
+				<li>
+					{$mastheadUser['user']->getFullName()|escape},
```

Comment:
Don't hard-code the `,` and order of presentation; put them into a locale key, or better yet, place them in separate `div` / `span` tags and use CSS to arrange them relative to each other. (Got a good example of how you'd like to see this, @jardakotesovec?)

Reply:
I will then use this:
```
<ul id="commaList">
  <li>apple</li>
  <li>orange</li>
  <li>banana</li>
</ul>
```

So that later it can be styled like this:
```
#commaList {
  display: inline;
  list-style: none;
}
#commaList li {
  display: inline;
}
#commaList li:after {
  content: ", ";
}
#commaList li:last-child:after {
    content: "";
}
```

And because we need to style the template however, we can change if needed...